ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -65,14 +104,25 @@ public static boolean isPseudoColumn(final ColumnName columnName) {
   }
 
   public static Set<ColumnName> pseudoColumnNames() {
-    return PSEUDO_COLUMN_NAMES;
+    return getPseudoColumnsFromVersion(CURRENT_PSEUDOCOLUMN_VERSION_NUMBER);
   }
 
   public static boolean isSystemColumn(final ColumnName columnName) {
     return systemColumnNames().contains(columnName);
   }
 
   public static Set<ColumnName> systemColumnNames() {
-    return SYSTEM_COLUMN_NAMES;
+    return SYSTEM_COLUMN_NAMES_CURRENT;
```

Comment:
This method should call the overload. We can also remove `SYSTEM_COLUMN_NAMES_CURRENT` if we do this.